app/views/eventRegistrations.html

```diff
@@ -30,31 +30,51 @@ <h2 class="page-title">Registrations</h2>
           <option value="over">Overpaid</option>
         </select>
       </div>
-      <div class="col-md-2 col-sm-7 stacked-spacing-col-sm">
+      <div class="col-md-2 col-sm-6">
         <label>Registrant type:</label>
         <select ng-model="queryParameters.filterRegType" ng-options="t.id as t.name for t in visibleFilterRegistrantTypes" class="form-control"></select>
       </div>
-      <div class="col-md-4 col-sm-5 text-right">
+      <div class="col-md-5 col-sm-6 text-right">
         <br>
         <div class="btn-group" role="group" dropdown auto-close="outsideClick">
           <button class="btn btn-default" ng-model="showMoreFilters" btn-checkbox>
             Filters
             <i class="fa fa-chevron-down" ng-show="!showMoreFilters"></i>
             <i class="fa fa-chevron-up" ng-show="showMoreFilters"></i>
           </button>
-          <button class="btn btn-primary" ng-click="refreshRegistrations()"><i class="fa fa-refresh"></i> Refresh</button>
+          <button class="btn btn-primary" ng-click="refreshRegistrations()">
+            <i class="fa fa-refresh"></i>
+            Refresh
+          </button>
           <button type="button" class="btn btn-primary dropdown-toggle" dropdown-toggle>
-            Columns
+            {{getScreenWidth() > 480 ? 'Display Columns' : 'Columns' }}
```

Comment:
You shouldn't have to use JS to do this. I would stick the word display in a `span` and make it hidden on xs. See http://getbootstrap.com/css/#responsive-utilities-classes.